test/val/val_barriers_test.cpp

```diff
@@ -245,16 +249,48 @@ OpControlBarrier %workgroup %workgroup %acquire_release_uniform_workgroup
   ASSERT_EQ(SPV_SUCCESS, ValidateInstructions(SPV_ENV_VULKAN_1_0));
 }
 
-TEST_F(ValidateBarriers, OpControlBarrierWebGPUSuccess) {
+TEST_F(ValidateBarriers, OpControlBarrierWebGPUAcquireReleaseSuccess) {
   const std::string body = R"(
-OpControlBarrier %workgroup %queuefamily %none
 OpControlBarrier %workgroup %workgroup %acquire_release_uniform_workgroup
 )";
 
   CompileSuccessfully(GenerateWebGPUShaderCode(body), SPV_ENV_WEBGPU_0);
   ASSERT_EQ(SPV_SUCCESS, ValidateInstructions(SPV_ENV_WEBGPU_0));
 }
 
+TEST_F(ValidateBarriers, OpControlBarrierWebGPURelaxedSuccess) {
+  const std::string body = R"(
+OpControlBarrier %workgroup %workgroup %uniform_workgroup
```

Comment:
When I read the links WebGPU issues, it sounds to me like only AcquireRelease should be supported.

From the 2019.04.08 notes:
>Consensus to have acquire/release barriers only, no relaxed ones.

Reply:
Thanks for the catch on that. I will update the PR.